x-pack/qa/saml-idp-tests/build.gradle

```diff
@@ -38,6 +38,9 @@ task setupPorts {
         idpMetaFile.write(content.toString(), "UTF-8")
     }
 }
+// Don't attempt to run integTest when Docker is not available
+integTest.enabled = idpFixtureProject.postProcessFixture.enabled
```

Comment:
With #40297 the test fixture plugin will do this automatically, but I think you're  problem is with `setupPorts` because the plugin doesn't know about that and it will run when there's no docker and thus no ports, so I think that also needs to be disabled here. 

For a more generic solution I think we need the plugin to add a `postProcessFixture` task on the projects that use the fixture, and properly disable it when there's no docker, so you could add a `doLast` to it here instead of defining a new task.

Reply:
>  so I think that also needs to be disabled here.

Great, I didn't see that in #40297 , will change this solution